devdocs/settingUp.md

```diff
@@ -94,14 +94,15 @@ Important: When a version is specified, please install that version instead of t
     Firefox 38.0.5 (latest release as at 7th June 2015) is supported.
    
 2. Before running the test suite, both the server and the test environment 
-   should be using the UTC time zone. Here is the procedure.
+   should be using the UTC time zone. The server and the test environment should 
+   also serve the CDN files locally instead. Here is the procedure.
```

Comment:
`CDN files` may not be self explanatory to a new dev. Elaborate some more. Explain at the level of an average CS2103 student.